mesonbuild/modules/qt.py

```diff
@@ -49,7 +49,11 @@ def _detect_tools(self, env, method):
         # What kind of an idiot thought that was a good idea?
         for compiler, compiler_name in ((self.moc, "Moc"), (self.uic, "Uic"), (self.rcc, "Rcc")):
             if compiler.found():
-                stdout, stderr = Popen_safe(compiler.get_command() + ['-v'])[1:3]
+                #Workaround since there is no easy way to know which tool/version support which flag
```

Comment:
__[Flake8]__

block comment should start with '# '


<!-- Comment by [SideCI](https://sideci.com) -->